src/pybind/mgr/dashboard/services/iscsi_config.py

```diff
@@ -48,13 +49,24 @@ def __init__(self):
 
 class IscsiGatewaysConfig(object):
     @classmethod
-    def _load_config(cls):
-        if OrchClient.instance().available():
-            raise ManagedByOrchestratorException()
+    def _load_config_from_store(cls):
         json_db = mgr.get_store(_ISCSI_STORE_KEY,
                                 '{"gateways": {}}')
         return json.loads(json_db)
 
+    @staticmethod
+    def _load_config_from_orchestrator():
+        config = {'gateways': {}}
+        try:
+            instances = OrchClient().list_service_info("iscsi")
+            for instance in instances:
+                config['gateways'][instance.nodename] = {
```

Comment:
After https://github.com/ceph/ceph-iscsi/pull/84 is merged we should change this code to query `{instance.service_url}/api/sysinfo/hostname` in order to get the correct key for 'gateways' dict (host short name or fqdn depending on the ceph-iscsi config).

But this can be fixed in a separate PR after https://github.com/ceph/ceph-iscsi/pull/84 is merged.